app/src/androidTest/java/org/mozilla/focus/activity/ContextMenusTest.kt

```diff
@@ -115,6 +115,22 @@ class ContextMenusTest {
         }
     }
 
+    @SmokeTest
+    @Test
+    fun shareLinkTest() {
+        val tab1Page = getGenericTabAsset(webServer, 1)
+        val tab2Page = getGenericTabAsset(webServer, 2)
+
+        searchScreen {
+        }.loadPage(tab1Page.url) {
+            verifyPageContent("Tab 1")
+            longPressLink("Tab 2")
+            verifyLinkContextMenu(tab2Page.url)
+            clickShareLink()
+            verifyShareAppsListOpened()
```

Comment:
Should we back out afterwards to close the list?

Reply:
It closes with the app at the end of the test. There are other tests with the same flow.